services/avsregistry/avsregistry_fake.go

```diff
@@ -0,0 +1,72 @@
+package avsregistry
+
+import (
+	"context"
+	"errors"
+	"math/big"
+
+	blsoperatorstateretrievar "github.com/Layr-Labs/eigensdk-go/contracts/bindings/BLSOperatorStateRetriever"
+	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
+	"github.com/Layr-Labs/eigensdk-go/types"
+)
+
+type FakeAvsRegistryService struct {
+	operators map[types.BlockNum]map[types.OperatorId]types.OperatorAvsState
+}
+
+func NewFakeAvsRegistryService(blockNum types.BlockNum, operators []types.TestOperator) *FakeAvsRegistryService {
```

Comment:
Where is this used?

Reply:
used in all the `blsagg_test.go` test cases. I'm using this instead of a mock, as we discussed.